ui/app/models/clients/new-init-activity.js

```diff
@@ -0,0 +1,11 @@
+import Model, { attr } from '@ember-data/model';
+// ARG TODO copied from before, modify for what you need
```

Comment:
keeping comment as this is still true and will modify depending on the future work with the flattening of the object data.